dev-tools/src/lambdas/dynamoOperations/dynamoDbGet.ts

```diff
@@ -16,11 +16,6 @@ export const dynamoDbGet = async (operationParams: OperationParams) => {
     })
   }
 
-  logger.info(
```

Comment:
For these logs that show the function parameter... is this not useful information? In my mind this is how you would identify which group of logs is relevant to your invocation. E.g. zendesk id, etc.